### ISSUE TITLE
Settings will disappear with time

### ISSUE THREAD

**Issue by nmi-brad** [form-o-fill/form-o-fill-chrome-extension#99]
Hello,

I've experienced an issue multiple times where chrome is closed for extended periods of time (unknown durations), but when i open up chrome and go to use form-o-fill I will lack my setting overrides. For example I want it always to show the popup as well as not to fire events after the fields are filled, those checkmarks are gone. I have users who are using the extension as well and seeing the same thing were their setting changes are removed/lost with time.

We are all running the latest chrome and latest form-o-fill versions on windows 10.

Let me know if there is anything else I can supply for this issue.

**Comment by thenoseman** [comment]
Hi!

Do you use remote rules import?

**Comment by nmi-brad** [comment]
Nope. Only a single rule is used (aside from the default) and was populated through the rule editor.

**Comment by thenoseman** [comment]
Ok, can you please access the [background page](https://github.com/form-o-fill/form-o-fill-chrome-extension/wiki#how-to-access-the-background-page) of the extension?

In the now open DevTools do a `state.optionSettings` in the console. 

It should look something like this:
```
{alwaysShowPopup: true, jpegQuality: "70", reevalRules: true, matchOnLoad: true, dontMatchOnTabSwitch: true, …}
```

Please do this when you have just set all options correctly and once the extension has forgotten the options!

What do you see?

**Comment by nmi-brad** [comment]
Thanks, due to the randomness i don't have an easy way to replicate so i'll notate that with values set correctly I see;
`{alwaysShowPopup: true, jpegQuality: "70", reevalRules: false, matchOnLoad: false, dontMatchOnTabSwitch: false, …}`

I'll alert my users so that if/when they see this appear again i'll get the forgotten values and post it up.

**Comment by nmi-brad** [comment]
One of my users complained again this morning, here is the output. (rather curious)
![Submission Image](https://s1.postimg.org/3qalnhpu2n/image.png)

**Comment by thenoseman** [comment]
Well that means the settings are gone :(
Another question: Did that user import the rules that day before the settings went missing?

**Comment by nmi-brad** [comment]
Nope, this is the same user I have been working with, he's not computer smart (hence the form filler), I confirmed he hasn't touched anything. I actually go the same thing to happen right after fixing the issue. I did nothing more than set the values, wait for the Saved message to appear and then closed the form-o-fill tab. I went back to the tab he was trying to fill out and the settings had seemingly been removed again (as clicking the FoF tab didn't give the dropdown we were expecting..)
